src/main/java/uk/gov/beis/els/util/ControllerUtils.java

```diff
@@ -72,4 +73,9 @@ public static void validateRatingClassIfPopulated(String legislationId, String e
     }
   }
 
+  public static void validateInternetLabelColour(String selectedLegislationId, SelectableLegislationCategory postMarch2021LegislationCategory, BindingResult bindingResult) {
+    if(postMarch2021LegislationCategory == null || postMarch2021LegislationCategory.getId().equals(selectedLegislationId)) {
```

Comment:
Can get rid of this null check if we do the validation group change mentioned earlier.